mix.exs

```diff
@@ -80,7 +80,8 @@ defmodule Quantum.Mixfile do
       {:excoveralls, "~> 0.5", only: [:dev, :test], runtime: false},
       {:inch_ex, "~> 0.5", only: [:dev, :docs], runtime: false},
       {:dialyxir, "~> 0.5", only: [:dev, :test], runtime: false},
-      {:credo, "~> 0.7", only: [:dev, :test], runtime: false}
+      {:credo, "~> 0.7", only: [:dev, :test], runtime: false},
+      {:persistent_ets, "~> 0.1.0"},
```

Comment:
Make the dependency optional. Not everyone will need the storage.

Reply:
Just an observation: here and in many many places, diligent coders have to specify `runtime: false` as they don't need the application to start (sometimes there is no application defined in the package). Just wondering if it was really a good move to make the dependencies start as an app by default? Kind of a move towards the newcomers, but in about a week or so, they become aware of the apps and become concerned about the `runtime: false` options as all the others do.
For me, the old good explicit aplication start looked a more neat and manture option.

I'll do the rebase.

Reply:
@pyatkov I'd rather not move away from the standard in Elixir. In my professional opinion, it will be even more complicated if every other place does this differently.

Reply:
@maennchen 
As I said, it was **Just an observation**., so ...
BTW, this is my personal accountm whose repo I use for developing the PR.